code/lib/core-server/src/index.ts

```diff
@@ -1,11 +1,7 @@
 // eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="./typings.d.ts" />
 
-export {
-  getPreviewHeadTemplate,
-  getPreviewBodyTemplate,
-  getPreviewMainTemplate,
```

Comment:
Is removing the main template a breaking change? Does it need a MIGRATION note? Or am I misreading this?

Reply:
They are functions for getting the template. They were shared between webpack4 & webpack5 builder.
Now that we only have 1 of those, I moved the code to the only builder that uses it.